cms-adapter/src/Rsbc.Dmf.CaseManagement.Service/Services/CaseService.cs

```diff
@@ -611,16 +739,25 @@ public async override Task<GetDocumentsReply> GetDriverDocuments(DriverLicenseRe
                         DocumentType = item.DocumentType ?? string.Empty,
                         DocumentTypeCode = item.DocumentTypeCode ?? string.Empty,
                         DocumentUrl = item.DocumentUrl ?? string.Empty,
-                        FaxReceivedDate = Timestamp.FromDateTimeOffset(item.FaxReceivedDate),
-                        ImportDate = Timestamp.FromDateTimeOffset(item.ImportDate),
+                      
+                        ImportDate = Timestamp.FromDateTimeOffset(item.ImportDate.Value),
```

Comment:
See above - need to check for null

Reply:
Fixed all the null checks for the FaxReceivedDate and Import Dates